src/coreclr/vm/comutilnative.cpp

```diff
@@ -1309,6 +1309,61 @@ extern "C" void QCALLTYPE GCInterface_AddMemoryPressure(UINT64 bytesAllocated)
     END_QCALL;
 }
 
+extern "C" void QCALLTYPE GCInterface_EnumerateConfigurationValues(QCall::StringHandleOnStack nameHandle, QCall::StringHandleOnStack valueHandle, EnumerateConfigurationValuesCallback callback)
+{
+    QCALL_CONTRACT;
+
+    BEGIN_QCALL;
+    GCInterface::EnumerateConfigurationValues(nameHandle, valueHandle, callback);
+    END_QCALL;
+}
+
+struct ConfigurationvaluesEnumerationContext
+{
+    QCall::StringHandleOnStack* nameHandle;
+    QCall::StringHandleOnStack* valueHandle;
+    EnumerateConfigurationValuesCallback callback;
+};
+
+void ConfigurationValue(void* context, const char* name, int type, int64_t intValue, const char* stringValue, bool boolValue)
+{
+    ConfigurationvaluesEnumerationContext* configurationContext = (ConfigurationvaluesEnumerationContext*)context;
+    configurationContext->nameHandle->Set(name);
+    if (type == 1)
```

Comment:
It would be better to define an enumeration for these types instead of sprinkling numbers around. Should be enough to have one in managed and one in native and reference each other.